### PR TITLE
feat: (bank-sdk) Add possibility to terminate bank-sdk from outer app

### PR DESCRIPTION
**Pull request by GeraltRiv**
PM-127

### CODE REVIEW

**Review by llevente — CHANGES_REQUESTED** [review]
The logic looks okay. Only a small documentation typo needs to be fixed.

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt [hunk 1]
```diff
@@ -616,4 +625,14 @@ object GiniBank {
     private fun releaseTransactionDocsFeature(context: Context) {
         giniBankTransactionDocs = null
     }
+
+    /**
+     *
+     *  Kills Bank sdk and stos Gini payment flow
```
[the remark this answers is not in the document]
Comment (GeraltRiv) [reply bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt]: Fixed

**Review by abolfazlimahdi — APPROVED** [review]
Looks good. Thank you!

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt [hunk 2]
```diff
@@ -628,7 +628,7 @@ object GiniBank {
 
     /**
      *
-     *  Kills Bank sdk and stos Gini payment flow
+     *  Stops Bank sdk and stos Gini payment flow
```
Comment (obaidgini) [inline bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt:631]: I think this typo is still there which was pointed out by lenci, which is "stos" in this line "Stops Bank sdk and this word -> stos Gini payment flow"

**Review by obaidgini — CHANGES_REQUESTED** [review]

**Review by obaidgini — APPROVED** [review]
Looks good to me now, thanks :)

**Pull request merged**